### ISSUE TITLE
NVDA reads spaces as blank lines in some html form elements.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1674]
**Reported by ashleycox on 2011-07-14 19:19**
In some html form elements, particularly textareas, NVDA reads spaces as blank lines. This can be reproduced in the textarea for submitting a new ticket on this site, in the google search box, and many other text input fields on many websites.

**Comment by ehollig** [comment]
Annoyingly, like https://github.com/nvaccess/nvda/issues/1674#issuecomment-155291675, I can not reproduce the bug using latest NVDA and IE 11. I tried using the textarea for this ticket, and the Google Search edit box. Using the arrow keys to review the text, and deleting the text, it read the spaces as expected.

**Comment by Adriani90** [comment]
I cannot reproduce this issue either in NVDA 2019.2 RC2. Closing as works for me. If someone can reproduce it, please coment on it and we can reopen the discussions. Thanks.
